Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -241,8 +253,17 @@ router.beforeEach(async (to, from, next) => {
         // Make sure that the route accepts the current state
         const currentUserState = calculateUserState();
         logger.debug(`current state: ${currentUserState}`);
+        const config = store.getters["config/webClient"];
```

Comment:
no need all the config settings, just modules

Reply:
updated